core/src/main/java/org/testcontainers/dockerclient/DockerClientConfigUtils.java

```diff
@@ -31,28 +29,31 @@
                                                       .exec(loggingCallback)
                                                       .awaitStarted();
                             loggingCallback.awaitCompletion(3, SECONDS);
-                            return loggingCallback.toString();
+                            return StringUtils.trimToNull(StringUtils.trim(loggingCallback.toString()));
                         } catch (Exception e) {
                             log.warn("Can't parse the default gateway IP", e);
                             return null;
                         }
                     }
-            ))
-            .map(StringUtils::trimToEmpty)
-            .filter(StringUtils::isNotBlank);
+            ));
+
+    @Deprecated
+    @Getter(lazy = true)
+    private static final Optional<String> detectedDockerHostIp = IN_A_CONTAINER ? getDefaultGateway() : Optional.empty();
```

Comment:
could you please put this method upper where it was, so that the diff will be smaller?